templates/kubernetes/compare.html

```diff
@@ -16,494 +20,502 @@
+              </td>
+              <td data-heading="Rancher">
+                <i class="p-icon--success-grey">yes</i>
+              </td>
+            </tr>
+            <tr>
+              <td data-heading="Category">Bare metal deployment</td>
+              <td data-heading="Canonical Kubernetes">
+                <i class="p-icon--success-grey">yes</i>
+              </td>
+              <td data-heading="Red Hat Openshift">
+                <i class="p-icon--success-grey">yes</i>
+              </td>
+              <td data-heading="Rancher">
+                <i class="p-icon--success-grey">yes</i>
+              </td>
+            </tr>
+            <tr>
+              <td data-heading="Category">Architectures supported</td>
+              <td data-heading="Canonical Kubernetes">X86, POWER, ARM, Z, s390x</td>
+              <td data-heading="Red Hat Openshift">X86, POWER, Z</td>
+              <td data-heading="Rancher">X86</td>
+            </tr>
+            <tr>
+              <td data-heading="Category">Pricing</td>
+              <td data-heading="Canonical Kubernetes">$$</td>
+              <td data-heading="Red Hat Openshift">$$$$$</td>
+              <td data-heading="Rancher">$$$</td>
+            </tr>
+          </tbody>
+        </table>
       </div>
-      <div class="col-4 u-align--center u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/338124d1-shields-security.svg",
-                alt="Security shields",
-                width="255",
-                height="150",
-                hi_def=True,
-                loading="auto",) | safe
-        }}
+      <div class="col-start-large-7 col-6">
+        <hr class="p-rule--muted" />
+        <i class="u-text--muted">Some categories are rated out of 5 for easy comparison.</i>
       </div>
     </div>
   </section>
 
-  <section class="p-strip">
-    <div class="row u-equal-height">
-      <div class="col-4 u-vertically-center u-hide--medium u-hide--small">
-        {{ image(url="https://assets.ubuntu.com/v1/d399db60-save-money.svg",
-                alt="Pricing",
-                width="220",
-                height="220",
-                hi_def=True,
-                loading="auto",) | safe
-        }}
-      </div>
-      <div class="col-8">
-        <h2>Canonical Kubernetes pricing</h2>
-        <p class="p-heading--4">Predictable pricing model. No licence fees. Enterprise support.</p>
-        <p>
-          Canonical's support services for Kubernetes follow a per host pricing model to bring you the best price-performance. Just count the number of physical or virtual machines you want support for and get coverage through <a href="/pro">Ubuntu Pro</a>, without any additional licence or support fees.
-        </p>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule--muted" />
+      <div class="col">
+        <h2>Multi-cloud deployments &amp; operations</h2>
```

Comment:
To be changed according to copydoc
```suggestion
        <h2>Multi-cloud deployments and operations</h2>
```